src/Momento.Sdk/Internal/VectorIndexDataClient.cs

```diff
@@ -154,14 +130,30 @@ public async Task<GetItemMetadataBatchResponse> GetItemMetadataBatchAsync(string
         }
     }
 
+    /// <summary>
+    /// Convert a list of ids to a filter expression requiring the item id to be in the set.
+    /// </summary>
+    /// <param name="ids"></param>
+    /// <returns></returns>
+    private static _FilterExpression idsToFilterExpression(IEnumerable<string> ids)
+    {
+        return new _FilterExpression
+        {
+            IdInSetExpression = new _IdInSetExpression()
+            {
+                Ids = { ids }
+            }
+        };
+    }
```

Comment:
Previously we allowed to select items to delete with a list of ids. We now have an id in set filter expression which is equivalent.